### PR TITLE
Add stateMachines CRaSH command

### PR DESCRIPTION
**Pull request by kasiastreich**
PR reopened from bitbucket.

Basic view of state machines running on the node. It doesn't have progress tracker.

View from demobench: 
screenshot from 2017-05-25 13-25-57

### CODE REVIEW

File: node/src/main/kotlin/net/corda/node/shell/StateMachinesShellUI.kt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            toStream.print(table)
+            toStream.println("Waiting for completion or Ctrl-C ... ")
+            toStream.flush()
+        }
+    }
+
+    @Synchronized
+    override fun onError(e: Throwable) {
+        toStream.println("Observable completed with an error")
+        e.printStackTrace()
```
Comment (mikehearn) [inline node/src/main/kotlin/net/corda/node/shell/StateMachinesShellUI.kt]: Hm, is this right? Printing a stack trace will break the table rendering, no?

Comment (kasiastreich) [inline node/src/main/kotlin/net/corda/node/shell/StateMachinesShellUI.kt]: Removed.

**Comment by mikehearn** [comment]
Really nice work! I pushed a commit with a few minor tweaks, nothing major. Now I try it, I see that it's actually more like a "tail" command isn't it, rather than a dashboard of what's running. Perhaps it should be called "flow follow" or "flow tail" instead .... sorry to be constantly changing my mind on this.

**Comment by mikehearn** [comment]
LGTM. Let's get it merged. Thanks! Good work!

**Pull request merged**